spec/lib/submission_builder/financial_transaction_spec.rb

```diff
@@ -2,14 +2,17 @@
 
 describe SubmissionBuilder::FinancialTransaction do
   describe '.build' do
+    let(:intake) { create(:state_file_az_owed_intake) }
+    let(:kwargs) { {} }
+    let(:submission) { create(:efile_submission, data_source: intake) }
+    let(:xml) {
+      Nokogiri::XML::Document.parse(
+        described_class.build(
+          submission, validate: false, kwargs: kwargs
+        ).document.to_xml)
+    }
```

Comment:
refactored tests in FinancialTransaction (I was here accidentally, thought I was adding stuff here but turned out i was supposed to be adding to AuthenticationHeader 😓)